app/pods/components/code-editor-component/template.hbs

```diff
@@ -5,9 +5,10 @@
   @allowedLanguages={{contest.allowedLanguages}}
   @onRunTask={{submission.runCodeTask}} 
   @onSubmitTask={{submitCodeTask}} 
-  @disablePaste={{fullScreen}}
+  {{!-- @disablePaste={{fullScreen}} --}}
+  @allowPaste={{contest.allowPaste}}
   @storageKey={{storageKey}}
-  @submission={{copySubmission}}
+  @copiedSubmission={{copiedSubmission}}
```

Comment:
let it be `@submission` as `<CodeWindow />` is unaware of copy flow. it just shows whatever is passed to it as submission